.gitlab-ci.yml

```diff
@@ -99,6 +99,18 @@ publish:release:
     - git fetch --depth=1 origin master
     - ./gradlew :dd-sdk-android:bintrayUpload
 
+publish:release-fresco:
+  tags: [ "runner:docker", "size:large" ]
```

Comment:
You need to rebase on master and apply the same changes as for the other tasks.

Reply:
yup already did that while you were reviewing ;)